CMSgov/easi-app
# Easi 3833/add system intake status to admin statuses

# EASI-3833

## Changes and Description

- Update Calculated Status to include expired and required
- Update translations to include language for requester and admin to show that an LCID was expired or an LCID was retired



## How to test this change
1. Bring up the app
2. Make an intake, (or seed intakes)
3. Issue an LCID
4. Retire the LCID
    a. ensure the status says Retired LCID: {{LCID}} 
5. Expire the LCID
    a. ensure the status says Expired LCID: {{LCID}}
6. Verify the status is the same on the admin actions pages
7. Verify the status is correct in both the requester, and admin tables 

## PR Author Review Checklist



## PR Reviewer Guidelines
- It's best to pull the branch locally and test it, rather than just looking at the code online!
- Check that all code is adequately covered by tests - if it isn't feel free to suggest the addition of tests.
- Always make comments, even if it's minor, or if you don't understand why something was done.
